hibernate-core/src/main/java/org/hibernate/Transaction.java

```diff
@@ -39,6 +39,16 @@
 	TransactionStatus getStatus();
 
 	/**
+	 * Indicate whether a resource transaction is in progress.
+	 *
+	 * @param isMarkedRollbackConsideredActive whether to consider {@link TransactionStatus#MARKED_ROLLBACK} as active.
+	 *
+	 * @return boolean indicating whether transaction is in progress
+	 * @throws HibernateException if an unexpected error condition is encountered.
+	 */
+	boolean isActive(boolean isMarkedRollbackConsideredActive);
```

Comment:
Not from what I can tell, just an oversight on my part, will move it to TransactionImplementor.